implementation/libs/jbb-lib-properties/src/main/java/org/jbb/lib/properties/FreshInstallPropertiesCreator.java

```diff
@@ -25,6 +29,18 @@
         this.resolver = resolver;
     }
 
+    private static void buildCompletePropertyFile(File propertyFile) {
```

Comment:
Probably, Apache Commons Properties which is used to read/write properties files would throw some exception. I can invoke lock/unlock methods on PropertiesConfiuration